code/obj/critter/drone.dm

```diff
@@ -1325,3 +1325,186 @@
 				src.health = 0
 				src.CritterDeath()
 
+/obj/critter/gunbot/drone/buzzdrone/sawfly // For associated objects check sawfly.dm in the obj folder
```

Comment:
/obj/critter/ makes me very sad. 

Please consider using /mob/living/critter/

See #8348 for details.